bundle/language_en.properties

```diff
@@ -1635,3 +1635,105 @@ angal.xmpp.usersinfo.border
+angal.anamnesis.smoke.yes.txt                                                                          = Yes
+angal.anamnesis.taxcode.txt                                                                            = Tax Number ID:
+angal.anamnesis.title.txt                                                                              = Patient History Edit
+angal.anamnesis.insertpatientexamination.tooltip                                                       = Insert Patient Examination
+angal.anamnesis.savethepatienthistory.tooltip                                                          = Save the Patient History
+angal.anamnesis.open.anamnesis.btn                                                                     = Anamnesis
+angal.anamnesis.title.txt                                                                              = Anamnesis
+angal.anamnesis.open.cardiovascolaris.txt                                                              = Cardiovascolaris
+angal.anamnesis.open.diseases.txt                                                                      = Diseases
+angal.anamnesis.open.drugsaddiction.txt                                                                = Drug Addiction
+angal.anamnesis.open.endocrinometabol.txt                                                              = Endocrine / Metabolic
+angal.anamnesis.open.gyno.txt                                                                          = Gyno/Obstetrical
+angal.anamnesis.open.hypertension.txt                                                                  = Hypertension
+angal.anamnesis.open.infective.txt                                                                     = Infective
+angal.anamnesis.open.neoplastic.txt                                                                    = Neoplastic
+angal.anamnesis.open.nothingtodeclare.txt                                                              = Nothing to declare
+angal.anamnesis.open.orto.txt                                                                          = Ortoreumatologic
+angal.anamnesis.open.other.txt                                                                         = Other
+angal.anamnesis.open.recentproblems.border                                                             = Recent Problems (Open)
+angal.anamnesis.open.remarks.border                                                                    = Remarks (Open)
+angal.anamnesis.open.respiratory.txt                                                                   = Respiratory
+angal.anamnesis.open.tooltip.excludenothingtodeclare.txt                                               = Exclude 'Nothing to declare'
+angal.anamnesis.open.tooltip.resetrecentproblems.txt                                                   = Reset Recent problems                                                       = Anamnesis
+angal.opd.anamnesis.btn.key                                                                            = M
```

Comment:
what's wrong here?

Reply:
Line 1738 has two equal signs; `= Reset Recent problems` and further to the right ` = Anamnesis`.

BTW, the tooltip text should not capitalize `Recent`; that is `Reset Recent problems` -> `Reset recent problems`